osdk/tools/docker/run_container.sh

```diff
@@ -7,11 +7,6 @@ set -e
 SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 ASTER_ROOT_DIR=${SCRIPT_DIR}/../..
 VERSION=$( cat ${ASTER_ROOT_DIR}/VERSION )
-
-if [ "$1" = "intel-tdx" ]; then
-    IMAGE_NAME="asterinas/osdk:${VERSION}-tdx"
-else
-    IMAGE_NAME="asterinas/osdk:${VERSION}"
-fi
+IMAGE_NAME="asterinas/osdk:${VERSION}"
```

Comment:
Leave this change out of this PR because the QEMU installed in this Docker image is still incompatible with TDX.